### PR TITLE
GA chat message preview, chat viewpoint, hide/unhide chat APIs, mark chat as read/unread APIs

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: api-reference/v1.0/api/chat-markchatunreadforuser.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+``` http
+POST https://graph.microsoft.com/v1.0/chats/19:7d898072-792c-4006-bb10-5ca9f2590649_8ea0e38b-efb3-4757-924a-5f94061cf8c2@unq.gbl.spaces/markChatUnreadForUser
+Content-Type: application/json
+Content-length: 158
+
+{
+  "user": {
+    "id" : "d864e79f-a516-4d0f-9fee-0eeb4d61fdc2"
+    "tenantId": "2a690434-97d9-4eed-83a6-f5f13600199a"
+  },
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chat-markchatunreadforuser.md:75]:
```suggestion
    "id" : "d864e79f-a516-4d0f-9fee-0eeb4d61fdc2",
    "tenantId": "2a690434-97d9-4eed-83a6-f5f13600199a"
  },
```

File: api-reference/v1.0/api/chat-get.md [hunk 2]
```diff
@@ -363,6 +285,10 @@ Content-type: application/json
     "chatType": "meeting",
     "webUrl": "https://teams.microsoft.com/l/chat/19%3Ameeting_YDZlYTYxOWUtYzdlMi00ZmMxLWIxMTAtN2YzODZlZjAxYzI4%40thread.v2/0?tenantId=2432b57b-0abd-43db-aa7b-16eadd115d34",
     "tenantId": "2432b57b-0abd-43db-aa7b-16eadd115d35",
```
Comment (RamjotSingh) [inline api-reference/v1.0/api/chat-get.md:287]: Add example of expanding lastMessagePreview

**Review by RamjotSingh — APPROVED** [review]

File: api-reference/beta/api/chat-get.md [hunk 3]
```diff
@@ -423,3 +423,79 @@ Content-type: application/json
...
+### Example 5: Get the chat along with the preview of the last message sent in the chat
+#### Request
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_meeting_chat",
+  "sampleKeys": ["19:meeting_ZDZlYTYxOWUtYzdlMi00ZmMxLWIxMTAtN2YzODZlZjAxYzI4@thread.v2"]
+}-->
+```msgraph-interactive
```
Comment (isvargasmsft) [inline api-reference/beta/api/chat-get.md:436]:
```suggestion
```http
```

File: api-reference/v1.0/api/chat-get.md [hunk 4]
```diff
@@ -375,6 +301,82 @@ Content-type: application/json
...
+### Example 5: Get the chat along with the preview of the last message sent in the chat
+#### Request
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_meeting_chat",
+  "sampleKeys": ["19:meeting_ZDZlYTYxOWUtYzdlMi00ZmMxLWIxMTAtN2YzODZlZjAxYzI4@thread.v2"]
+}-->
+```msgraph-interactive
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chat-get.md:313]:
```suggestion
```http
```

File: api-reference/v1.0/resources/chat.md [hunk 5]
```diff
@@ -78,6 +78,7 @@ A chat is a collection of [chatMessages](chatmessage.md) between one or more par
 | Relationship | Type |Description|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/chat.md:78]: Please add the new property viewPoint to the properties table and the JSON representation of this file. In addition, please add it to the examples of methods that return the chat resource in the response body. In addition, please add the new property to the examples in the Get chat topic.

File: api-reference/v1.0/resources/chatmessageinfo.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+doc_type: resourcePageType
+---
+
+# chatMessageInfo resource type
+
+Namespace: microsoft.graph
+
+Represents a preview of a [chatMessage](../resources/chatmessage.md) resource. This object can only be fetched as part of a list of [chats](../resources/chat.md).
+
+## Properties
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/chatmessageinfo.md:16]: This is an entity type, hence it requires a section for Methods. 
```suggestion

Namespace: microsoft.graph

Represents a preview of a [chatMessage](../resources/chatmessage.md) resource. This object can only be fetched as part of a list of [chats](../resources/chat.md).

## Methods
None.

## Properties
```

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @jessiechan27, 

Thank you for your work in this PR. Very complete overall. 

Please add the missing property I indicated in the comments and arrange the properties table in alphabetical order. 

Feel free to ping me on the side if you have questions. 

Kind regards, 

Isaac

**Review by isvargasmsft — APPROVED** [review]

**Comment by Lauragra** [comment]
@jessiechan27 , if you would like to publish this PR, please remove the **do not merge** label and add **ready to merge**.

**Pull request merged**